com.ibm.streamsx.topology/opt/python/include/splpy.h

```diff
@@ -28,7 +29,51 @@
  * Functionality for executing Python within IBM Streams.
  */
 
+#if PY_MAJOR_VERSION == 3
+#define DECL_PYTHON_LIBNAME std::string pyLib("libpython3.5m.so");
```

Comment:
Minor point, but this approach leaves the code non-intuitive. Since it's just `DECL_PYTHON_LIBNAME ` so it's not obvious it's defining the variable pyLib.

Seems it would be simpler to just have the #define set the name of the library, and then have the actual code be:

`std::string pyLib(TOPOLOGY_PYTHON_LIBNAME);`